pages/api/locations/read.ts

```diff
@@ -12,15 +12,18 @@ export default async function handler(
       if (typeof req.body === 'string') {
         req.body = JSON.parse(req.body)
       }
-      const { id: I } = req.body.data
-
+      const { id: I } = req.body
       const id: number = I as number
       let location = null
 
       if (id) {
         location = await prisma.location.findUniqueOrThrow({
           where: {
             id: id
+          },
+          include: {
+            Prescriptions: true,
```

Comment:
If we can already use the perscriptions /read endpoint to retrieve perscriptions by locationId, how come you want to include it here too?